src/main/java/com/exadel/discount/config/SpringFoxConfig.java

```diff
@@ -1,28 +0,0 @@
-package com.exadel.discount.config;
-
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-import org.springframework.web.bind.annotation.RestController;
-import springfox.documentation.builders.ApiInfoBuilder;
-import springfox.documentation.builders.PathSelectors;
-import springfox.documentation.builders.RequestHandlerSelectors;
-import springfox.documentation.spi.DocumentationType;
-import springfox.documentation.spring.web.plugins.Docket;
-
-@Configuration
-public class SpringFoxConfig {
-    @Bean
-    public Docket api() {
-        return new Docket(DocumentationType.OAS_30)
-                .select()
-                .apis(RequestHandlerSelectors.withClassAnnotation(RestController.class))
-                .paths(PathSelectors.any())
-                .build()
-                .apiInfo(new ApiInfoBuilder()
-                        .title("Discount API")
-                        .description("API for the Discount Application")
-                        .version("1.0")
-                        .build()
-                );
-    }
-}
```

Comment:
Why did you delete this file? Maybe it's a mistake in merging

Reply:
Is it back now?